### PR TITLE
Email case insensitivity

### PR DESCRIPTION
**Pull request by aditiii23**

### CODE REVIEW

File: frontend/src/components/Signup/Signup.js [hunk 1]
```diff
@@ -31,6 +31,7 @@ const Signup = () => {
     const error = {}
     const emailRegex = /^[^\s+@]+@[^\s@]+\.[^\s@]{2,}$/i
     const phoneRegex = /^\+?\d{1,3}[- ]?\d{3}[- ]?\d{3}[- ]?\d{4}$/
+    values.email = values.email.toLowerCase()
```
Comment (mrpriyansh) [inline frontend/src/components/Signup/Signup.js]: What if the account is created using the API directly(i.e. without frontend)? Does your code cover that case?

**Review by mrpriyansh — CHANGES_REQUESTED** [review]

File: frontend/src/components/Signup/Signup.js (hunk 1 above)
Comment (aditiii23) [reply frontend/src/components/Signup/Signup.js]: Updated changes for this case in backend

Comment (mrpriyansh) [reply frontend/src/components/Signup/Signup.js]: If you send ADITI@gmail.com from frontend without lowercasing it what will be saved into the db?

Comment (aditiii23) [reply frontend/src/components/Signup/Signup.js]: it will be saved as aditi@gmail.com

File: backend/controllers/userController.js [hunk 2]
```diff
@@ -7,11 +7,11 @@ const User = require("../model/user.model")
 
 const registerUser = async (req, res, next) => {
   try {
-    const { name, email, password, confirmpwd, phone } = req.body
+    let { name, email, password, confirmpwd, phone } = req.body
```
Comment (mrpriyansh) [inline backend/controllers/userController.js]: only email value is changing!! other values can be const

Comment (aditiii23) [reply backend/controllers/userController.js]: done

File: backend/controllers/userController.js [hunk 3]
```diff
@@ -43,7 +44,8 @@ const registerUser = async (req, res, next) => {
 
 const loginUser = async (req, res, next) => {
   try {
-    const { email, password } = req.body
+    let { email, password } = req.body
```
Comment (mrpriyansh) [inline backend/controllers/userController.js]: Still there is let

**Review by mrpriyansh — APPROVED** [review]

**Pull request merged**